content/en/docs/components/training/prometheus.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "Prometheus Monitoring"
```

Comment:
Please can you add this guide under user guides ?

Reply:
Sure. Have moved this guide under user guides section.